### ISSUE TITLE
Update to use Json.Net

### ISSUE THREAD

**Issue by clintonrocksmith** [EodHistoricalData/EODHistoricalData.NET#19]
It is a performance boost to move to Json.Net rather than Newtonsoft.Json

Is this something that you would like me to do?

**Comment by deios0** [comment]
Yes, we will be happy to improve our library this way, please let me know if you need any assistance.

**Comment by DatapulseEOD** [comment]
For anyone looking for a simple source of clean NSE EOD BhavCopy data, I share the ready-to-use files. The format is plain text OHLC with volume, tested specifically with Amibroker and MetaStock so you can import directly without extra conversion. You can grab the daily NSE EOD BhavCopy now.

https://tinyurl.com/4wm85vyb

[2026-03-19-NSE-EQ.txt](https://github.com/user-attachments/files/26242468/2026-03-19-NSE-EQ.txt)
[2026-03-20-NSE-EQ.txt](https://github.com/user-attachments/files/26242466/2026-03-20-NSE-EQ.txt)
[2026-03-23-NSE-EQ.txt](https://github.com/user-attachments/files/26242467/2026-03-23-NSE-EQ.txt)
[2026-03-24-NSE-EQ.txt](https://github.com/user-attachments/files/26242469/2026-03-24-NSE-EQ.txt)
